### DISCUSSION TITLE
Does this works for Gitlab as well?

### DISCUSSION THREAD

**Discussion by basitqayoom** [dlvhdr/gh-dash#373] [General]
I am using Gitlab extensively for my work, that's why!

**Comment by Omnikron13** [comment]
Pretty sure it's hooked into the gh cli and APU, but there is a gitlab clli set of tools and API as far as I know? Probably possible to port the UI onto GitLab but you'd have to go spelunk the the source code. Node sure how engtangled it is with GH.

**Reply by adrianlungu** [reply]
It's not built with provider independence it seems, tried to do a bit of digging into the code, and it is possible to make it independent, but it's not gonna be easy work. 

Not sure if the maintainers are looking into doing that at the moment though
